changelog.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
What is this file for? A kind of configuration?
This should be placed in another directory then.